lightwood/encoder/audio/mfcc.py

```diff
@@ -9,20 +9,40 @@
 
 class MFCCEncoder(BaseEncoder):
     """
-    Audio encoder. Uses `librosa` to compute the Mel-frequency spectral coefficients (MFCCs) of the audio file. They are a common feature used in speech and audio processing. The features are a 2D array, flattened into a 1D one.
+    Audio encoder.
+
+    Uses `librosa` to compute the Mel-frequency spectral coefficients (MFCCs) of the audio file. They are a common feature used in speech and audio processing.
+
+    The output feature for any given audio file is a 2D array, flattened into a 1D one to comply with the expected format in lightwood mixers.
+
+    Note that this encoder does not have a .decode() method. As such, models that predict audio as output are not supported at this time.
     """  # noqa
     is_trainable_encoder: bool = False
 
     def __init__(self, is_target: bool = False):
+        """
+        :param is_target: whether this encoder's column is the target. Should be false as encoder is not bi-directional. 
+        """  # noqa
+        assert not is_target
         super().__init__(is_target)
 
     def prepare(self, priming_data: pd.Series):
+        """
+        Retrieves and stores the expected output_size.
```

Comment:
"The audio encoder undergoes rule-based processing. Thus, the prepare statement only returns the output dimension size"

is the output dimension size *fixed* always? as in, no matter what input audio goes in, do we expect the same size vector? cause if this is the case we can literally just so self.output_size = number

Reply:
It _is_ fixed, but inside `.encode()`. I agree this should be refactored to go for best practices, but I'd argue it should be on a separate PR.